eox_hooks/receivers.py

```diff
@@ -0,0 +1,81 @@
+"""This module defines the signals receivers for defined hooks."""
+import logging
+from importlib import import_module
+
+from django.conf import settings
+
+from eox_hooks.constants import Status
+from eox_hooks.models import HookExecutionAudit
+
+log = logging.getLogger(__name__)
+
+DEFAULT_TASK_MODULE = "eox_hooks.tasks"
+DEFAULT_TASK_NAME = "default_task"
+
+
+def hooks_handler(sender, **kwargs):
```

Comment:
is this mean that we will have only **one** receiver method ???

Reply:
yes! that's the plan